evm/src/cpu/kernel/tests/ripemd.rs

```diff
@@ -1,56 +0,0 @@
-use anyhow::Result;
-use ethereum_types::U256;
-use itertools::Itertools;
-
-use crate::cpu::kernel::aggregator::KERNEL;
-use crate::cpu::kernel::interpreter::run_interpreter;
-
-fn make_input(word: &str) -> Vec<u32> {
-    let mut input: Vec<u32> = vec![word.len().try_into().unwrap()];
-    input.append(&mut word.as_bytes().iter().map(|&x| x as u32).collect_vec());
-    input.push(u32::from_str_radix("deadbeef", 16).unwrap());
-    input
-}
-
-#[test]
-fn test_ripemd_reference() -> Result<()> {
```

Comment:
Was this also just for debugging?

Reply:
yes; this was my original test before it got replaced by importing the reference implementation.